src/presets/netlify.ts

```diff
@@ -85,3 +75,62 @@ export const netlifyEdge = defineNitroPreset({
     }
   }
 })
+
+async function writeRedirects (nitro: Nitro) {
+  const redirectsPath = join(nitro.options.output.publicDir, '_redirects')
+  let contents = '/* /.netlify/functions/server 200'
+
+  for (const [key, value] of Object.entries(nitro.options.routes).filter(([_, value]) => value.redirect)) {
+    const redirect = typeof value.redirect === 'string' ? { to: value.redirect } : value.redirect
+    // TODO: update to 307 when netlify support 307/308
+    contents = `${key.replace('/**', '/*')}\t${redirect.to}\t${redirect.statusCode || 301}\n` + contents
```

Comment:
Defaults (301 fallback) shall be done directly in nitro. For fixing 307/308 not supported, we need to **check** for them.